robocorp-code/bin/create_env/robot.yaml

```diff
@@ -7,3 +7,8 @@ tasks:
 artifactsDir: output
 
 condaConfigFile: conda.yaml
+environmentConfigs:
+  - environment_windows_amd64_freeze.yaml
```

Comment:
I wasn't aware of this, so, this is mostly a question from me trying to discover how this works... does this mean that on windows/amd64 it will use `environment_windows_amd64_freeze.yaml` instead of `conda.yaml`?

Reply:
Yes, this is a quite fresh feature in RCC that it will allow multiple *_freeze.yaml files, which get prioritized from the top until a suitable platform match is found. In case no platform matches RCC uses the given condaConfigFile.

Just a nice way to wrap multi platform dependencies into one robot.yaml